charts/azure/azure-disk-loss/engine.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: nginx-chaos
+spec:
+  # It can be active/stop
+  engineState: 'active'
+  chaosServiceAccount: azure-disk-loss-sa
+  # It can be delete/retain
+  jobCleanUpPolicy: 'retain'
```

Comment:
we can remove `jobCleanUpPolicy: 'retain'` as it is by default retain - just to maintain uniformity.